src/index.html

```diff
@@ -1,13 +1,45 @@
-<!doctype html>
+  <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="true">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main class="card" data-qa="card">
+      <section class="section1 card__main">
+        <img src="./images/imac.jpeg" alt="photo of product" class="section1__image">
+        <h2 class="section1__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+      </section>
+
+      <section class="section2 card__code">
+        <h3 class="section2__code">Код товара: 195434</h3>
+      </section>
+
+      <section class="section3 card__stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <span class="section3__reviews">Отзывов: 5</span>
+      </section>
+
+      <section class="section4 card__price">
+        <span class="section4__text">Цена:</span>
+        <span class="section4__number">69 999 грн</span>
+      </section>
+
+      <a href="/" data-qa="hover"><button class="card__button">Купить</button></a>
```

Comment:
do you need `button`? move class to link - get the same
in fact, this is not a button, although it looks like this. it's a link to product